i18n/italian.json

```diff
@@ -423,218 +423,218 @@
+    "music-ploff-disabled" : "L'autoplaylist è stata disattivata!",
+    "music-plnow-no-pl" : "L'autoplaylist è stata disattivata! Usa {} per caricare una autoplaylist!",
+    "music-plnow-now-user-auto" : "L'autoplaylist corrente è quella di {}'.\nAutoplaylist: `{}/{}`",
+    "music-plnow-now-user" : "L'autoplaylist corrente è quella personale di {}'.",
+    "music-plnow-now-auto" : "L'autoplaylist corrente è quella di {}. \nAutoplaylist: `{}/{}`",
+    "music-plnow-now" : "L'autoplaylist corrente è {} !",
+    "music-plstart-loaded-user" : "L'autoplaylist personale di {} è stata caricata!",
+    "music-plstart-your-loaded" : "La tua autoplaylist personale è stata caricata!",
+    "music-plstart-loaded" : "L'autoplaylist {} è stata caricata!",
+    "music-plstart-dont-have" : "Mi spiace! Questo utente non ha una personale autoplaylist.",
+    "music-plstart-doesnt-exists" : "L'autoplaylist chiamata {} non esiste! Controlla il tuo messaggio o usa {} se vuoi crearla.",
+    "music-plnew-special-char" : "Per favore non usare caratteri speciali.",
+    "music-plnew-try-to-hack" : "Non puoi creare una autoplaylist personale come questa, specialmente se non è il tuo ID.",
+    "music-plnew-already-exists" : "{} già esiste! Mi spiace...",
+    "music-plnew-created" : "Nuova autoplaylist {} creata!",
+    "music-plinfo-not-found" : "Non riesco a trovare questa autoplaylist..Mi spiace!",
+    "music-pl-disabled" : "L'autoplaylist è al momento disabilitata! Usa {} per caricarne.",
+    "music-plinfo-loading" : "Sto calcolando questa autoplaylist, attendere prego...",
+    "music-plinfo-empty-auto" : "Non ci sono canzoni in questa autoplaylist, usa {} per aggiungerne qualcuna!",
+    "music-plinfo-not-accessible" : "NON E' ACCESSIBILE, RIMUOVILA CON {}",
+    "music-plinfo-perso-display" : "La playlist personale di {} ha {} canzoni !",
+    "music-plinfo-display" : "L'autoplaylist {} ha {} canzoni !",
+    "music-plinfo-warnings" : "Attenzione : Ci sono {} link inesistenti nella tua autoplaylist. Puoi rimuoverli tutti con {}",
+    "music-plinfo-warning" : "Attenzione : C'è un link inesistente nella tua autoplaylist. Puoi rimuoverlo facilmente con {}",
+    "music-plinfo-too-much" : "L'autoplaylist ha troppe canzoni per essere visualizzate qui. Ti invierò il contenuto dell'autoplaylist in DM in pochi secondi.",
+    "music-plrepair-someone" : "Puoi solo modificare le tue autoplaylist personali.",
+    "music-plrepair-loading" : "Sto riparando l'autoplaylist, attendere prego...",
+    "music-plrepair-all-ok" : "E' tutto ok, questa autoplaylist non necessita di essere riparata !",
+    "music-plrepair-removed-links" : "Completato, {} link inesistenti sono stati rimossi dall'autoplaylist.",
+    "music-plrepair-removed-link" : "Completato, `1` link inesistente è stato rimosso dall'autoplaylist.",
     "music-lyrics-title-not-find" : "Cannot find the title, please ensure your URL is correct.",
```

Comment:
Ohh and this one too please

Reply:
Ok ;)

Reply:
I edited directly as Propose change, sorry for creating duplicate PR

Reply:
Np